advanced/Scripts/webpage.sh

```diff
@@ -568,6 +569,52 @@ RemoveDHCPStaticAddress() {
 
 }
 
+AddDHCPStaticHostname() {
+    echo -e "  ${TICK} Adding static DHCP Hostname entry..."
+
+    mac="${args[2]}"
+    ip="${args[3]}"
+    host="${args[4]}"
+    reload="${args[5]}"
+
+    validHost="$(checkDomain "${host}")"
+    if [[ -n "${validHost}" ]]; then
+        if valid_ip "${ip}" || valid_ip6 "${ip}" ; then
+            # Mac address will be used as a key to support RemoveDHCPStaticHostname
+            printf "#%s\n%s %s\n" "${mac}" "${ip}" "${validHost}" >> "${dhcpstatichostnames}"
```

Comment:
Did you check if this is actually working? I'm not sure if `dnsmasq` accepts ab `addn-hosts` file with a syntax other than `ip hostname`.

Reply:
Also consider `addKey()` from `utils.sh`

Reply:
>  I'm not sure if dnsmasq accepts ab addn-hosts file with a syntax other than ip hostname.  
Yes, the format dnsmasq uses is the same as entries in [/etc/hosts](https://linux.die.net/man/8/dnsmasq)
I tested this by cherry picking this change on the master branch
Addition/romoval works as expected. Existing entries in static leases are unaffected by this change.
> Also consider addKey() from utils.sh  
This suggestion is for adding the new addn-hosts, correct? I will test & update the PR then

Reply:
> Yes, the format dnsmasq uses is the same as entries in [/etc/hosts](https://linux.die.net/man/8/dnsmasq)
I tested this by cherry picking this change on the master branch
Addition/removal works as expected. Existing entries in static leases are unaffected by this change.

It's not working if you add the `MAC`.

```
Aug  8 17:00:39 dnsmasq[14014]: bad address at /etc/pihole/custom.list line 17
Aug  8 17:00:39 dnsmasq[14014]: read /etc/pihole/custom.list - 16 addresses

rockpi@rockpi-4b:~$ wc -l /etc/pihole/custom.list 
17 /etc/pihole/custom.list

rockpi@rockpi-4b:~$ sed -n 17p /etc/pihole/custom.list 
80:86:F2:9C:57:16 1.2.3.4 test
```

Reply:
@yubiuser did you add this entry manually to this file?
The entry should look like:
```
#80:86:F2:9C:57:16
1.2.3.4 test
```

Reply:
Sorry, sure. I missed the `#`